lab-jr/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Missing a few key pieces of your package.json. Mainly description, full author name, git repo info, and dependencies.